drag_and_drop_v2/public/js/drag_and_drop_edit.js

```diff
@@ -31,7 +31,7 @@ function DragAndDropEditBlock(runtime, element, params) {
                     _fn.tpl = {
                         zoneInput: Handlebars.compile($("#zone-input-tpl", element).html()),
                         zoneElement: Handlebars.compile($("#zone-element-tpl", element).html()),
-                        zoneDropdown: Handlebars.compile($("#zone-dropdown-tpl", element).html()),
+                        zoneCheckbox: Handlebars.compile($("#zone-checkbox-tpl", element).html()),
```

Comment:
Absolutely correct, except this particular template only renders a single checkbox. Similarly, `zoneDropdown`, when it was there, only rendered a single selection item. So, I think `zoneCheckbox` in the singular is the better fit in this particular case; the duplication of `zoneCheckbox` for each zone is handled around line 365.